src/braze/tags/connectedContent.ts

```diff
@@ -70,10 +70,20 @@ export default <ITagImplOptions>{
       }
     }
 
-    let body
+    let body = this.options.body
     if (this.options.body) {
-      body = (await this.liquid.parseAndRender(this.options.body, ctx.getAll())).replace(/(?:\r\n|\r|\n|)/g, '')
+      if (method.toUpperCase() === 'POST' && contentType.toLowerCase().includes('application/json')) {
+        const jsonBody = {}
+        for (const element of this.options.body.split('&')) {
+          const bodyElementSplit = element.split('=')
+          jsonBody[bodyElementSplit[0]] = (await this.liquid.parseAndRender(bodyElementSplit[1], ctx.getAll())).replace(/(?:\r\n|\r|\n|)/g, '')
```

Comment:
hmm, feels better to strip `\n` or `\r` earlier before we enter this loop?

Reply:
We need to `parseAndRender` the body first, as liquid persists those characters during variable capture, which is what we really need to strip off. If we parse and render first, then we can't be sure there won't be an additional `&` rendered to mess up this split.